src/explore-education-statistics-admin/src/pages/admin-dashboard/components/PublicationSummary.tsx

```diff
@@ -167,7 +168,10 @@ const PublicationSummary = ({ publication, onChangePublication }: Props) => {
             <td colSpan={2} data-testid={`Releases for ${publication.title}`}>
               {releases.length > 0 ? (
                 <ul className="govuk-list">
-                  {releases.filter(noAmendmentInProgressFilter).map(release => (
+                  {orderBy(
+                    releases.filter(noAmendmentInProgressFilter),
+                    rel => rel.title,
```

Comment:
I might be wrong because I don't know what the title is in this case but would have thought Releases want ordering by the back end by Year and and then Time Period Coverage, for example to avoid this when sorted alphabetically

```
2019 Week 1
2019 Week 12
2019 Week 2
2019 Week 27
2019 Week 3
2019 Week 34
2019 Week 4
```

Reply:
This is an issue. Talked to Ben about this, and just for the sake of clearing up the Jira board, I'll move the ticket with this issue back, but leave the PR as it is, and then fix it on its own new PR.

Ben told me that I should have made a PR for each ticket and he was right.